core/src/main/java/io/jenkins/pluginhealth/scoring/model/updatecenter/Plugin.java

```diff
@@ -31,8 +31,31 @@
 
 public record Plugin(String name, VersionNumber version, String scm,
                      ZonedDateTime releaseTimestamp, List<String> labels,
-                     int popularity, String requiredCore, String defaultBranch) {
+                     int popularity, String requiredCore, String defaultBranch, List<IssueTrackers> issueTrackers) {
+
+    /* This builder function is used in test cases when List<IssueTrackers> is not required as a mandatory parameter. */
+    public static Plugin of(String name, VersionNumber version, String scm,
+                            ZonedDateTime releaseTimestamp, List<String> labels,
+                            int popularity, String requiredCore, String defaultBranch) {
+        return new Plugin(name, version, scm, releaseTimestamp,
+            labels, popularity, requiredCore, defaultBranch, List.of());
+    }
+
     public io.jenkins.pluginhealth.scoring.model.Plugin toPlugin() {
         return new io.jenkins.pluginhealth.scoring.model.Plugin(this.name(), this.version(), this.scm(), this.releaseTimestamp());
     }
+
+    public List<IssueTrackers> getIssueTrackers() {
+        return this.issueTrackers;
+    }
+
```

Comment:
You don't need this on records. "Getters" are created by default. The name of the method is `issueTrackers()` like the field specified in the record definition. See https://docs.oracle.com/en/java/javase/17/language/records.html

```suggestion

```